sql/catalyst/src/main/scala/org/apache/spark/sql/connector/catalog/CatalogV2Implicits.scala

```diff
@@ -143,7 +142,7 @@ private[sql] object CatalogV2Implicits {
     }
   }
 
-  private lazy val catalystSqlParser = new CatalystSqlParser(SQLConf.get)
+  private lazy val catalystSqlParser = new CatalystSqlParser()
```

Comment:
We can directly use `object CatalystSqlParser` in this file and remove this lazy val, as `CatalogV2Implicits` is already an object.

Reply:
done